backend/models/course.model.ts

```diff
@@ -27,6 +29,14 @@ const CourseSchema: Schema = new Schema({
     type: [String],
     required: true,
   },
+  private : {
+    type: Boolean,
+    default: false,
+  } ,
```

Comment:
Please run the linter and push a new commit with the changes it makes. If you have any concerns/questions, lmk

Reply:
Done :)